autopilot/core/terminal.py

```diff
@@ -123,6 +120,12 @@ def __init__(self):
         # store instance
         globals()['_TERMINAL'] = self
 
+        # Load settings
+        # These are stored in ~/.config/Autopilot/Terminal.conf
+        # Currently, the only setting is "geometry", but loading here
+        # in case we start to use other ones in the future
+        self.settings = QtCore.QSettings("Autopilot", "Terminal")        
```

Comment:
I like this. let's move it to be in the autopilot directory so it's all in one place? looks like `QSettings` supports being constructed with a filename, i'll add a pref for that

Reply:
will add description to docstring too